packages/framework/ai-collab/package.json

```diff
@@ -0,0 +1,141 @@
+{
+	"name": "@fluid-experimental/ai-collab",
+	"version": "2.4.0",
+	"description": "Experimental package to simplify integrating AI into Fluid-based applications",
+	"homepage": "https://fluidframework.com",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/FluidFramework.git",
+		"directory": "packages/framework/ai-collab"
+	},
+	"license": "MIT",
+	"author": "Microsoft and contributors",
+	"sideEffects": false,
+	"type": "module",
+	"exports": {
+		".": {
+			"import": {
+				"types": "./lib/public.d.ts",
+				"default": "./lib/index.js"
+			},
+			"require": {
+				"types": "./dist/public.d.ts",
+				"default": "./dist/index.js"
+			}
+		},
+		"./alpha": {
```

Comment:
Should we add `beta` as well? We're generating a beta report.

Reply:
I don't know that it should be getting generated, but I haven't figured out why it is. We don't have any beta APIs, nor npm scirpts that reference beta...

Reply:
I figured out we can tweak which models are generated in `api-extractor.json`, but I'm leaning towards not doing it, just for consistency with most other packages. This wouldn't be the first instance of a package that doesn't have a `/beta` export but is currently producing a report file for the beta API level. Most of them look like that, I think.